### DISCUSSION TITLE
Custom module > Advanced Module = no quick create

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15721] [General] [answered]
Converted module to advanced and this module is not showing in Fields - Quick Create, not any fields in Layout Editor can be "ticked" as Quick Create. Am I missing something?

Best

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
not a bug, advanced modules do not support Quick Create modal

**Reply by s2b-git** [reply]
Understood. Easy workaround is to create shortcut menu
